cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperatorMockTest.java

```diff
@@ -266,21 +266,6 @@ public void testCreateUpdateDelete(TestContext context) {
             updateAsync.complete();
         });
         updateAsync.await();
-        LOGGER.info("Reconciling again -> delete");
```

Comment:
There is an assertion we could write for the deletion case, but it would need the `MockKube` to understand the owner references.

Reply:
I was thinking it would checks that we really are setting the owner reference on all the necessary resources. It's only of marginal value, and now I know about subtleties of the PVC deletion, I guess it's more trouble than it's worth.